README.md

```diff
@@ -222,6 +222,21 @@ Airbrake.configure do |c|
 end
 ```
 
+#### versions
+
+A hash of arbitrary versions that your application uses that you want to
+track (Rails, Linux kernel, API version). By default, it's empty.
+
+```ruby
+Airbrake.configure do |c|
+  c.dependency_versions = {
+    'Rails' => '5.2.0',
+    'bundler' => '1.16.1',
+    'Linux kernel' => '4.4.0-97-generic'
```

Comment:
It would be nice to remove spaces and use either `Linux_kernel` or `LinuxKernel` so it is easier to use in aggregations like `context.versions.LinuxKernel`. I am not even sure aggregations support spaces in keys.

Reply:
👍